### ISSUE TITLE
1. An address and contact field for liasion person/lead should be provided.

### ISSUE THREAD

**Issue by mstenta** [mstenta/farm_nfa#14]
> Groups
> 1. An address and contact field for liasion person/lead should be provided.

Related to #13 ? Curious to understand what the vision is for "Groups" in these ideas...

**Comment by LeScruf** [comment]
Assuming groups is used as a collective term for specific assets, we want to use this for creating 'projects' which have specific assets that are managed together. For example, NFA may have a client - such as the Uganda Electricity Distribution Company LtD (UEDCL) that wants to plant 1,000 ha of Eucalyptus for utility poles. This would then become a project, with a contact at UEDCL and a lead at NFA. Because a specific asset or set of assets (crops) are being managed as a group, this then becomes a 'Project'. Similarly for any other forest asset that is managed under a specific agreement or arrangement - for example, an NGO working with NFA to manage a specific nature forest reserve - the asset then would be that particular Forest Reserve? It's more accurate therefore to call groups 'projects' it would seem...

**Comment by mstenta** [comment]
Thanks for the clarification - I think that maybe "Groups" is not the right approach for this. We may want to create a dedicated "Project" entity type for this.

Adding the "Discovery" label so we can discuss this together.

**Comment by pcambra** [comment]
@mstenta do we have any plans on 2.x for this?

**Comment by LeScruf** [comment]
Yes - I think this still remains a useful / needed functionality...
